lib/spanner/config/semantic_validator.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Spanner.Config.SemanticValidator do
+
+  alias Piper.Permissions.Ast
+  alias Piper.Permissions.Parser
+
+  @moduledoc """
+  Validates bundle config semantics.
+  """
+
+
+  @doc """
+  Accepts a config map and validates semantics.
+  """
+  @spec validate(Map.t) :: :ok | {:error, [{String.t, String.t}]}
+  def validate(config) do
+    verify_rules(config)
+  end
```

Comment:
1. Rename `verify_rules/1` to `verify/1`.
1. Make `verify/1` public.
1. Drop `validate/1`.
1. Profit!